feat(content): Conversation about Greenrock's reversion to piracy (#8217)
endless-sky/endless-sky: data/human/free worlds side plots.txt

		conversation
			`You receive a message from Freya: "Hello again, Captain! Thanks for your help escorting the materials to Solace. Ever since we developed the Dreadnoughts people have been talking about how badly we need a generator that can meet their energy needs, and that's what Delta V Corporation has been working on. So, take a look next time you're on Solace or one of our other major manufacturing centers. Sorry for the mystery surrounding the cargo, but we didn't exactly want to advertise that a huge shipment of fissionable material was headed for Free Worlds space."`
				decline

mission "Greenrock Lost Announcement"
	invisible
	landing
	source
		government "Free Worlds"
	to offer 
		has "event: fw abandoned Greenrock"
	on offer
		conversation 

> 			`Waiting in <planet>'s spaceport for your ship to be refuelled, you hear an announcement coming from an unseen loudspeaker. "Attention! The Free Worlds have withdrawn our presence from Greenrock. Be advised that any further travel to the Shaula system may carry the risk of pirate attacks." The message cuts off abruptly, only to repeat a minute later.`

Reviewer: ```suggestion
			`While waiting in <planet>'s spaceport for your ship to be refuelled, you hear an announcement coming from an unseen loudspeaker. "Attention! The Free Worlds have withdrawn our presence from Greenrock. Be advised that any further travel to the Shaula system may carry the risk of pirate attacks." The message cuts off abruptly, only to repeat a minute later.`
```

			`	A militia captain who paused nearby to listen to the announcement shakes his head before recognizing you. "Well, that's a bloody shame, isn't it?"`
			choice

> 				`	"Yeah, I risked my life and my position taking Greenrock."`

Reviewer 2: ```suggestion
				`	"Yeah, I risked my life, and my position taking Greenrock."`
```

While this could just be a dialogue is allowed to be improper thing, I really think the pacing is best with one of these two options.

> 				`	"Yeah, I risked my life and my position taking Greenrock."`

Reviewer 2: ```suggestion
				`	"Yeah, I risked my life and position taking Greenrock."`
```

					goto risked
					to display
						has "FW Pirates: Attack 3: done"
				`	"It was probably going to happen sooner or later."`
				`	"We shouldn't have wasted the resources on it at all."`
			`	"I suppose, but things should have gone differently. The Free Worlds, and the Pact before that, was founded to eliminate piracy from the South, so I thought it was the right thing to do. We really put the pirates on the wire when we went straight for their guts. But now it looks like we'll have our hands full for the foreseeable future, and the pirates are sure going to make the most out of this."`
				goto end
			label risked

> 			`	"I was also at Shaula too, although it wasn't me who was commanding our ship. I thought my old captain did the right thing at that time, but he took the fall for it, same as you. It's too bad, we really put the pirates on the wire when we went straight for their guts. But now it looks like we'll have our hands full for the foreseeable future, and the pirates are sure going to make the most out of this."`

Reviewer: ```suggestion
			`	"I was at Shaula too, although it wasn't me who was commanding our ship. I thought my old captain did the right thing at that time, but he took the fall for it, same as you. It's too bad, we really put the pirates on the wire when we went straight for their guts. But now it looks like we'll have our hands full for the foreseeable future, and the pirates are sure going to make the most out of this."`
```

Reviewer: I think “our” could be changed to “the” but it’s not a definitely needed change.

> 			`	"I was at Shaula too, although it wasn't me who was commanding the ship. I thought my old captain did the right thing at that time, but he took the fall for it, same as you. It's too bad, we really put the pirates on the wire when we went straight for their guts. But now it looks like we'll have our hands full for the foreseeable future, and the pirates are going to make the most out of this for sure."`

Reviewer 2: `same as you.`

I feel like this specific bit should either be paired with an overall more bitter tone in regard to the consequences suffered from everyone who took that side in this answer (especially as a contrast to acceptance of the outcome) -or- be replaced with another wording because the player, in fact, did not suffer loss of ship or rank within the FW as a consequence.

Author: Hmm, maybe I could infer that there was some sort of informal 'purge' that began after Tomek started disobeying orders, but since the Free Worlds couldn't risk losing more ships than they already had, it wasn't ever allowed to go further than a few captains, and even they were brought back into the militia (like you).

I'm guessing the consequences were different for the player because they're a foreign 'mercenary' type that joined the Miltia without having grown up in it, while this guy and his captain were career militiamen.

Reviewer 2: Yea, something like cycling out or even demoting captains and officers, so they'd "lose" command of the ship but still be part of the Free Worlds forces would make a lot of sense. And keep it at a similar level to what the player faces but from the different perspective.

And sorry, I somehow missed this one at the time :)

	invisible
	landing
	source
		government "Free Worlds"
	to offer
		has "event: fw abandoned Greenrock"
	on offer
		conversation
			`While waiting in <planet>'s spaceport for your ship to be refuelled, you hear an announcement coming from an unseen loudspeaker. "Attention! The Free Worlds have withdrawn our presence from Greenrock. Be advised that any further travel to the Shaula system may carry the risk of pirate attacks." The message cuts off abruptly, only to repeat a minute later.`
			`	A militia captain who paused nearby to listen to the announcement shakes his head before recognizing you. "Well, that's a damn shame, isn't it?"`
			choice
				`	"Yeah, I risked my life and position taking Greenrock."`
					goto risked
					to display
						has "FW Pirates: Attack 3: done"
				`	"It was probably going to happen sooner or later."`
				`	"We shouldn't have wasted the resources on it at all."`
			`	"I suppose, but things should have gone differently. The Free Worlds, and the Pact before that, was founded to eliminate piracy from the South, so I thought it was the right thing to do. We really put the pirates on the wire when we went straight for their guts.`
			choice

> 				`	"Well, the most obvious threat to the Free Worlds' sovereignty is the Navy. Attacking Greenrock hasn't helped in that eliminating that threat."`

Author: ```suggestion
				`	"Well, the most obvious threat to the Free Worlds' sovereignty is the Navy. Attacking Greenrock hasn't helped in that regard."`
```

					goto challenge
				`	"Tomek's plan didn't work out, though. The other pirate planets didn't surrender when Greenrock fell, like he had wanted."`

> 				`	(Stay silent.)`
> 					goto silent

Author: ```suggestion
```

		conversation
			`You receive a message from Freya: "Hello again, Captain! Thanks for your help escorting the materials to Solace. Ever since we developed the Dreadnoughts people have been talking about how badly we need a generator that can meet their energy needs, and that's what Delta V Corporation has been working on. So, take a look next time you're on Solace or one of our other major manufacturing centers. Sorry for the mystery surrounding the cargo, but we didn't exactly want to advertise that a huge shipment of fissionable material was headed for Free Worlds space."`
				decline

> mission "Greenrock Lost Announcement"

Reviewer 3: ```suggestion
mission "FW Greenrock Lost Announcement"
```

	invisible
	landing
	source
		government "Free Worlds"
	to offer
		has "event: fw abandoned Greenrock"
	on offer
		conversation

> 			`While waiting in <planet>'s spaceport for your ship to be refuelled, you hear an announcement coming from an unseen loudspeaker. "Attention! The Free Worlds have withdrawn our presence from Greenrock. Be advised that any further travel to the Shaula system may carry the risk of pirate attacks." The message cuts off abruptly, only to repeat a minute later.`

Reviewer 3: ```suggestion
			`While waiting in <planet>'s spaceport for your ship to be refueled, you hear an announcement coming from an unseen loudspeaker. "Attention! The Free Worlds have withdrawn our presence from Greenrock. Be advised that any further travel to the Shaula system may carry the risk of pirate attacks." The message cuts off abruptly, only to repeat a few minutes later.`
```
